src/components/Switch/switch.component.css

```
@@ -0,0 +1,58 @@
+html {
+  font-size: 62.5%;
+}
+
+:root {
+  --main-color: #6200ee;
+  --secondary-color: #c39ef8;
+  --default-color: #fff;
+}
+
+.inputWrapper {
+  position: relative;
+  top: 0.5rem;
+  left: 0.5rem;
+}
+
+.slider {
+  display: flex;
+  cursor: pointer;
+  width: 7rem;
+  height: 3rem;
+  border-radius: 1.4rem;
+  background-color: #c6c6c6;
+  position: relative;
+  transition: background-color 0.2s;
+}
+
+.slider::before {
+  content: '';
+  position: absolute;
+  /* --"transition" no longer works if this is on and to and left are omitted*/
+  /* transform: translateY(-12%); */
+  top: -0.48rem;
+  left: -0.40rem;
+  width: 4rem;
+  height: 4rem;
+  border-radius: 9rem;
+  transition: 0.2s;
+  /* transition-duration: 900ms; */
+  background-color: var(--default-color);

```

```suggestion
  position: absolute;
  top: -0.48rem;
  left: -0.40rem;
  width: 4rem;
  height: 4rem;
  border-radius: 9rem;
  transition: 0.2s;
  background-color: var(--default-color);
```

This is good enough for acceptance, so we want to make sure no unwanted commented code is in the PR.